.github/workflows/sharing-smart-contracts-reusable.yml

```
@@ -54,61 +54,61 @@ jobs:
         working-directory: packages/sharing-smart-contract
         run: npm run uml
 
-      - name: Static Analyzer
-        uses: crytic/slither-action@v0.4.0
-        id: slither
-        with:
-          sarif: result.sarif
-          fail-on: none
-          target: 'packages/sharing-smart-contract/'
-
-      - name: Start Anvil
-        run: |
-          anvil --host 0.0.0.0 --port 8545 --hardfork berlin --fork-url https://bellecour.iex.ec --chain-id 134 --gas-limit 6700000 --gas-price 0 &
-
-      - name: Wait for Anvil to start
-        run: |
-          timeout=30
-          interval=1
-          echo "Waiting for Anvil to start..."
-          for ((i=0; i<timeout; i++)); do
-            if nc -z localhost 8545; then
-              echo "Anvil is operational."
-              exit 0
-            fi
-            echo "Attempt $((i+1)) of $timeout: Anvil is not ready, waiting ${interval}s..."
-            sleep $interval
-          done
-          echo "Error: Anvil did not start within the timeout period."
-          exit 1
-
-      - name: Upgrade test
-        working-directory: packages/sharing-smart-contract
-        run: npm run upgrade-local-fork -- --network local-bellecour-fork
-
-        # TODO check why the CI does not fail when the following error occurs
-        # (when the RPC node is not available):
-        # 1) Uncaught error outside test suite
-      - name: Hardhat tests
-        working-directory: packages/sharing-smart-contract
-        run: npm run test -- --network local-bellecour-fork
-
-      - name: Test deployment script
-        working-directory: packages/sharing-smart-contract
-        run: |
-          POCO_ADDRESS=0x3eca1B216A7DF1C7689aEb259fFB83ADFB894E7f \
-          DATASET_REGISTRY_ADDRESS=0x799DAa22654128d0C64d5b79eac9283008158730 \
-            npm run deploy -- --network local-bellecour-fork
-
-      - name: Set Directory Permissions
-        working-directory: packages/sharing-smart-contract
-        run: sudo chmod -R 777 .
-
-      - name: Forge Tests
-        working-directory: packages/sharing-smart-contract
-        run: forge test --no-match-test "invariant" -vvvv
-
-      - name: Upload SARIF file
-        uses: github/codeql-action/upload-sarif@v3
-        with:
-          sarif_file: ${{ steps.slither.outputs.sarif }}
+      # - name: Static Analyzer
```

Uncomment all stages before merge
```suggestion
      - name: Static Analyzer
```